arch/arc/Kconfig

```diff
@@ -370,6 +370,28 @@ endmenu
 config DCACHE_LINE_SIZE
 	default 32
 
+config DCACHE_REGION_OPERATIONS
+	bool "DCACHE region operations"
+	depends on CACHE_MANAGEMENT && DCACHE && CPU_HS4X
+	default y
+	help
+	  Perform L1 data cache management operations by regions rather than line by line in a loop,
+	  improves performance of cache management operations.
+
+config SLC
+	bool "System level cache"
+	depends on CACHE_MANAGEMENT && DCACHE && CPU_HS4X
+	default n
+	help
+	  This option enables System Level Cache, and adds SLC support to the data cache management operations.
+
+config SLC_LINE_SIZE
+	int "SLC line size"
+	depends on SLC
+	default 64
```

Comment:
For the hardware (design-time) the default L2 cache size is 128, not 64. (but ok to leave it 64 if you prefer).